.storybook/preview.js

```diff
@@ -1,2 +1,4 @@
-import "./stories.css"
+import "./stories.css";
```

Comment:
same as above. the style linter should complain here. I'll create a ticket for it